satellite/console/consoleweb/consoleql/mail.go

```diff
@@ -0,0 +1,56 @@
+// Copyright (C) 2019 Storj Labs, Inc.
+// See LICENSE for copying information
+
+package consoleql
+
+const (
+	// ActivationPath is key for pass which handles account activation
+	ActivationPath = "activationPath"
+)
+
+// AccountActivationEmail is mailservice template with activation data
+type AccountActivationEmail struct {
+	ActivationLink string
+}
+
+// Template returns email template name
+func (tmpl *AccountActivationEmail) Template() string {
```

Comment:
For such short things I usually make them single line. And since these don't need the receiver, then no need to define it.

```
// Template returns email template name
func (AccountActivationEmail) Template() string { return "welcome" }
```